Automattic/sensei
# Fix module status overlapping module name on single course page

Fixes #1330.

There's nothing quite like building a Sensei extension to discover how difficult it is to build a Sensei extension. 😆

This PR updates the module status and places it inside the element in which it visually appears, rather than having it inside a sibling element and then using absolute positioning to move it. 😱The use of absolute positioning makes it very difficult to place other elements into this area of the page.

## Testing
1. Create a course that has multiple lessons inside multiple modules. Ensure the module names are fairly long and that they have a description.
2. Start taking that course.
3. Complete all the lessons in a module so that the "Completed" module status appears on the single course page.
4. Complete one of the lessons in another module with multiple lessons so that the "In progress" module status appears on the single course page.
5. If your module names are long enough, you should see the module name wrap to a new line instead of running into the module status. If your module names are not long enough, you can narrow the size of the browser window.
6. Click on a module name and ensure that the module status is styled correctly.
7. Test on multiple themes.

## Screenshots

### Storefront
#### Before
![storefront-before](https://user-images.githubusercontent.com/1190420/58288225-f28b1d80-7d80-11e9-8e5f-54449c5a605c.jpg)

#### After
![storefront-after](https://user-images.githubusercontent.com/1190420/58288239-ffa80c80-7d80-11e9-87e0-e8f9ae38298a.jpg)

### Divi
#### Before
![divi-before](https://user-images.githubusercontent.com/1190420/58288252-0a62a180-7d81-11e9-8e30-7adcb1a28de9.jpg)

#### After
![divi-after](https://user-images.githubusercontent.com/1190420/58288264-13537300-7d81-11e9-8d1a-c881d4c03e0c.jpg)
